MetaMask/metamask-extension
# TypeScript tooling fixes

## Explanation
This introduces a few tiny changes to our existing tooling to work better with ts files.

1. tsconfig.json extended to ignore some of our build output directories, which cause typescript to slow down a fair bit and warn about excluding large folders.
2. jest.config.js extended to consider coverage of ts files, and to run .ts tests in more folders
3. verify-locale script extended to consider ts files in more locations.

## Manual Testing Steps
1. pull branch
2. `yarn`
3. run `yarn storybook` 
4. Ensure vscode isn't slow when in a .ts file (you can do this on a different branch to see what it looks like, or remove some of the tsconfig entries on this branch)



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
